packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
+| placement     | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                      | ✅       |
+| children      | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element            | ✅       |
+| isOpen        | `boolean`                                                                | `undefined` | Controls the tooltip state                                                |          |
+| defaultIsOpen | `boolean`                                                                | `undefined` | If true, the tooltip will be visible initially                            |          |
+| onDismiss     | `Callback`                                                               | `undefined` | Called when tooltip is closed, either by mouseout or users pressing `Esc` |          |
+
+### Examples:
+
+#### Usage
+
+```js
+// basic, uncontrolled
+<Tooltip content="Amount reversed to customer bank account" placement="top">
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+
+// controlled
+const [isOpen, setIsOpen] = React.useState(false);
+
+<Tooltip
+  onDismiss={() => setIsOpen(false)}
+  isOpen={isOpen}
+  content="Amount reversed to customer bank account"
+>
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+
+// default open
+<Tooltip defaultIsOpen={true} content="Amount reversed to customer bank account">
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>;
+```
+
+## Library
+
+We will be using [FloatingUI](https://floating-ui.com/) to position the tooltip & handle the basic tooltip logic. Floating UI supports both Web & ReactNative.
```

Comment:
Perhaps add few points on bundle size

Reply:
Sure will add. on web side with all the necessary elements it will be around 10kb 

https://bundlejs.com/?q=%40floating-ui%2Freact&treeshake=%5B%7B%0A++arrow%2Cflip%2CFloatingArrow%2Coffset%2CuseFloating%2CuseFocus%2CuseHover%2CuseInteractions%2CuseTransitionStyles%2C%7D%5D&config=%7B%22esbuild%22%3A%7B%22external%22%3A%5B%22react%22%2C%22react-dom%22%5D%7D%7D

This handles everything, collision aware placement, arrow, automatic animations,  offsets, interactions (hover, focus)